Make enumtables schema aware

ttung opened the pull request:
### Description
This library was unaware of schemas and as such choked whenever it came across a schema.

### Test plan
used in later PR.

phoenixAja commented on third-party/sqlalchemy-enum-tables/enumtables/alembic_autogen.py:
```diff
@@ -36,15 +36,19 @@ def compare_enums(autogen_context, upgrade_ops, schema_names):
+			if is_table_present(fully_qualified_table_name, autogen_context.connection):
+				items = {
+					r[0]
+					for r in autogen_context.connection.execute(
+						"SELECT item_id FROM {}".format(fully_qualified_table_name)
+					)
```
should we use bind parameters for this? 

`sql = sql.text("SELECT item_id FROM :tablename")`
`autogen_context.connection.execute(sql, tablename=fully_qualified_table_name)`

ttung replied:
I'll fix this in #59, as this code actually get yoinked there.

phoenixAja replied:
ok makes sense

phoenixAja commented on third-party/sqlalchemy-enum-tables/enumtables/alembic_ops.py:
```diff
@@ -7,53 +7,64 @@
 		return
 	values = ', '.join("('" + v + "')" for v in operation.data)
-	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.tablename, vl = values)
+	txt = 'INSERT INTO {tn} (item_id) VALUES {vl}'.format(tn = operation.fully_qualified_tablename, vl = values)
```
same comment as above about bind parameters with these insert statements

ttung replied:
fixed, and ready for review.

Commit:
Make enumtables schema aware

### Description
This library was unaware of schemas and as such choked whenever it came across a schema.

### Test plan
used in later PR.
